packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js

```diff
@@ -23,7 +25,14 @@ class DurationEdit extends React.Component {
   }
 
   handleHourChange = e => {
-    const hours = e.target.value.replace(/[^\d]/g, '')
+    let hours = e.target.value.replace(/[^\d]/g, '')
+    if (this.maxHours) {
+      if (hours > this.maxHours) {
+        hours = 0
+      } else if (hours < 0) {
+        hours = this.maxHours
```

Comment:
is it possible to enter a negative value in case of no maxHours defined?

Reply:
Shower thoughts: can a duration be negative?

Reply:
Duration should not be negative. Lines 32/33 were redundant anyway because the regex above and the prop on the input element didn't allow negative values.